.github/workflows/Android-CI.yml

```diff
@@ -6,18 +6,11 @@ jobs:
   buildTest:
     name: Build & Test JDK ${{ matrix.java_version }}
     runs-on: macOS-latest
-    strategy:
-      matrix:
-        java_version: [1.8]
```

Comment:
What if the default version changes? Wouldn't this omission lead to possible version errors in the future?

Reply:
Who knows about the future ?
Currently all is fine, and I like to make things as simple as possible.

If it runs ever in an issue (maybe never), some action will be taken

Reply:
Hmm...I think this is counter productive as it allows for future errors that are completely unrelated to the code changed in the commit being tested by the CI. If the build depends on Java 1.8 and someone tries to build the same code a few years down the road and it breaks because the new runner defaults to Java 1.9, then this is non-deterministic behavior, which I thought you were not a fan of. 

Better summary of this argument [here](https://blog.danlew.net/2015/09/09/dont-use-dynamic-versions-for-your-dependencies/).

Reply:
You are somehow right, this repo has a long history and maybe will exists longer.
Ehh whatever, this commit could be also done in 5 years too, please see d84f8bb